### PR TITLE
RHDEVDOCS-2865: Document the changes in the Add Page and the ability to hide cards on it 

### PR DESCRIPTION
**Pull request by HarshCasper**
JIRA ID: [RHDEVDOCS-2865](https://issues.redhat.com/browse/RHDEVDOCS-2865)

Peer Reviewer: @Preeticp 

Aligned Team: DevTools

This PR is for the OCP 4.8 Release.

Preview link: https://deploy-preview-33988--osdocs.netlify.app/openshift-enterprise/latest/applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective

### CODE REVIEW

**Comment by Preeticp** [comment]
Please get SME/QE LGTM please.

File: modules/odc-importing-codebase-from-git-to-create-application.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ The following procedure walks you through the *From Git* option in the *Develope
 
 .Procedure
 
-. In the *+Add* view, click *From Git* to see the *Import from git* form.
+. In the *+Add* view, visit the *Git Repository* tile and click on the *From Git* to see the *Import from git* form.
```
Comment (Preeticp) [inline modules/odc-importing-codebase-from-git-to-create-application.adoc:14]: 'visit' is not the right word in this context.
Suggest: In the *+Add* view, click *From Git* in the *Git Repository* tile to see the *Import from git* form.

Comment (Preeticp) [reply modules/odc-importing-codebase-from-git-to-create-application.adoc:14]: Ideally we use click **button/option**, and not click on the button/option, unless you want to mention the type of field like, for example, click the **YAML** tab

File: modules/odc-using-the-developer-catalog-to-add-services-or-components.adoc [hunk 2]
```diff
@@ -9,8 +9,8 @@ You use the Developer Catalog to deploy applications and services based on Opera
 
 .Procedure
 
-. In the *Developer* perspective, navigate to the *+Add* -> *From Catalog* to view all the available services in the *Developer Catalog*.
-. Under *All Items*, select the kind of service or the component you need to add to your project. For this example, select *Databases* to list all the database services and then click *MariaDB* to see the details for the service.
+. In the *Developer* perspective, navigate to the *+Add* view and from the *Developer Catalog* tile, click on *All Services* to view all the available services in the *Developer Catalog*.
```
Comment (Preeticp) [inline modules/odc-using-the-developer-catalog-to-add-services-or-components.adoc:12]: click on **All Services**/click **All Services**

File: modules/odc-using-the-developer-catalog-to-add-services-or-components.adoc [hunk 3]
```diff
@@ -9,8 +9,8 @@ You use the Developer Catalog to deploy applications and services based on Opera
...
+. Under *All Services*, select the kind of service or the component you need to add to your project. For this example, select *Databases* to list all the database services and then click *MariaDB* to see the details for the service.
 +
 .Developer Catalog
```
Comment (Preeticp) [inline modules/odc-using-the-developer-catalog-to-add-services-or-components.adoc:15]: Did All Items change to *All Services*? If yes please remove the screenshot below, it shows All Items. Use screenshots only to explain very complicated user flows.

**Review by Preeticp — COMMENTED** [review]
@HarshCasper looks good overall, have left a few comments, please let me know if you have any questions.

File: modules/odc-using-the-developer-catalog-to-add-services-or-components.adoc (hunk 3 above)
Comment (HarshCasper) [reply modules/odc-using-the-developer-catalog-to-add-services-or-components.adoc:15]: Done!

File: modules/odc-using-the-developer-catalog-to-add-services-or-components.adoc (hunk 2 above)
Comment (HarshCasper) [reply modules/odc-using-the-developer-catalog-to-add-services-or-components.adoc:12]: Done!

File: modules/odc-importing-codebase-from-git-to-create-application.adoc (hunk 1 above)
Comment (HarshCasper) [reply modules/odc-importing-codebase-from-git-to-create-application.adoc:14]: Done!

File: modules/odc-deploying-java-applications.adoc [hunk 4]
```diff
@@ -5,9 +5,9 @@
 [id="odc-deploying-java-applications_{context}"]
 = Upload JAR files for easy deployment of Java applications
 
-You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications. You can upload a JAR file using the following options: 
+You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications. You can upload a JAR file using the following options:
 
-* Navigate to the *+Add* view of the *Developer* perspective, click the *Upload JAR file* tile through the *From Local Machine* option. Browse and select your JAR file, or drag and drop a JAR file to deploy your application.
+* Navigate to the *+Add* view of the *Developer* perspective, navigate to the *From Local Machine* tile and click on the *Upload JAR file* option. Browse and select your JAR file, or drag and drop a JAR file to deploy your application.
 
 * Navigate to the *Topology* view and use the *Upload JAR file* option, or drag and drop a JAR file to deploy your application.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply modules/odc-deploying-java-applications.adoc:13]: Done!

File: modules/odc-deploying-java-applications.adoc [hunk 5]
```diff
@@ -5,9 +5,9 @@
 [id="odc-deploying-java-applications_{context}"]
 = Upload JAR files for easy deployment of Java applications
 
-You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications. You can upload a JAR file using the following options: 
+You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications. You can upload a JAR file using the following options:
 
-* Navigate to the *+Add* view of the *Developer* perspective, click the *Upload JAR file* tile through the *From Local Machine* option. Browse and select your JAR file, or drag and drop a JAR file to deploy your application.
+* Navigate to the *+Add* view of the *Developer* perspective, navigate to the *From Local Machine* tile and click on the *Upload JAR file* option. Browse and select your JAR file, or drag and drop a JAR file to deploy your application.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply modules/odc-deploying-java-applications.adoc:10]: Done!

File: modules/odc-creating-sample-applications.adoc [hunk 6]
```diff
@@ -11,8 +11,8 @@ The following procedure explains the *Samples* option in the *Developer* perspec
 
 .Procedure
 
-. In the *+Add* view, click *Samples* to see the *Samples* page. 
-. On the *Samples* page, select one of the available sample applications to see the *Create Sample Application* form. 
+. In the *+Add* view, click on the *Samples* tile to see the *Samples* page. You can alternatively pick up a sample application from the *Getting started resources* as well.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply modules/odc-creating-sample-applications.adoc:14]: Done!

File: applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc [hunk 7]
```diff
@@ -7,14 +7,40 @@ toc::[]
...
+** *From Git*: Select a Git repository to import code to build and deploy an application.
+** *From Devfile*: Select a Git repository to import a Devfile to build and deploy an application.
+** *From Dockerfile*: Select a Git repository to import a Dockerfile to build and deploy an application.
+
+* *Container images*: Use existing images from an image stream or registry to deploy it on to the OpenShift Container Platform.
+
+* *Pipelines*: Use Tekton Pipeline to create CI/CD Pipelines for your software delivery process on the OpenShift Container Platform.
+
+* *Serverless*: Explore the *Serverless* options to create, build and deploy stateless and serverless applications on the OpenShift Container Platform.
+** *Channel*: Create a Knative Channel to create an event forwarding and persistence layer with in-memory and reliable implementations.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc:37]: It is not present on the User Interface as of now. So I don't think we would need to cover it.

File: applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc [hunk 8]
```diff
@@ -7,14 +7,40 @@ toc::[]
...
+* *Git repository*: Import an existing codebase from your Git repository to build and deploy an application on OpenShift Container Platform.
+** *From Git*: Select a Git repository to import code to build and deploy an application.
+** *From Devfile*: Select a Git repository to import a Devfile to build and deploy an application.
+** *From Dockerfile*: Select a Git repository to import a Dockerfile to build and deploy an application.
+
+* *Container images*: Use existing images from an image stream or registry to deploy it on to the OpenShift Container Platform.
+
+* *Pipelines*: Use Tekton Pipeline to create CI/CD Pipelines for your software delivery process on the OpenShift Container Platform.
+
+* *Serverless*: Explore the *Serverless* options to create, build and deploy stateless and serverless applications on the OpenShift Container Platform.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc:36]: Done!

File: applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc [hunk 9]
```diff
@@ -7,14 +7,40 @@ toc::[]
...
+====
+
+* *Git repository*: Import an existing codebase from your Git repository to build and deploy an application on OpenShift Container Platform.
+** *From Git*: Select a Git repository to import code to build and deploy an application.
+** *From Devfile*: Select a Git repository to import a Devfile to build and deploy an application.
+** *From Dockerfile*: Select a Git repository to import a Dockerfile to build and deploy an application.
+
+* *Container images*: Use existing images from an image stream or registry to deploy it on to the OpenShift Container Platform.
+
+* *Pipelines*: Use Tekton Pipeline to create CI/CD Pipelines for your software delivery process on the OpenShift Container Platform.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc:34]: Done!

File: applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc [hunk 10]
```diff
@@ -7,14 +7,40 @@ toc::[]
...
-* *Container Image*: Use existing images from an image stream or registry to deploy it on to {product-title}.
-* *From Dockerfile*: Import a dockerfile from your Git repository to build and deploy an application.
-* *YAML*: Use the editor to add YAML or JSON definitions to create and modify resources.
-* *From Catalog*: Explore the *Developer Catalog* to select the required applications, services, or source to image builders and add it to your project.
-* *Database*: See the *Developer Catalog* to select the required database service and add it to your application.
-* *Operator Backed*: Explore the *Developer Catalog* to select and deploy the required Operator-managed service.
-* *Helm Chart*: Explore the *Developer Catalog* to select the required Helm chart to simplify deployment of applications and services.
+* *Getting Started resources*: Use these resources to help you get started with the Console. You can choose to hide the header using the *Options* menu {kebab}.
+** *Creating applications using sample*: Use the existing code samples to get started with creating applications on the OpenShift Container Platform.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc:11]: Done!

File: applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc [hunk 11]
```diff
@@ -7,14 +7,40 @@ toc::[]
...
+** *Build with guided documentation*: Follow the guided documentation to build applications and familiarize yourself with key concepts and terminologies.
+** *Explore new developer features*: Explore the new features and resources within the *Developer* perspective.
+
+* *Developer catalog*: Explore the Developer Catalog to select the required applications, services, or source to image builders, and then add it to your project.
+** *All Services*: Browse the catalog to discover services across OpenShift Container Platform.
+** *Database*: Select the required database service and add it to your application.
+** *Operator Backed*: Select and deploy the required Operator-managed service.
+** *Helm chart*: Select the required Helm chart to simplify deployment of applications and services.
+** *Event Source*: Select an Event source to register interest in a class of events from a particular system.
+
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc:21]: Done!

File: applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc [hunk 12]
```diff
@@ -7,14 +7,40 @@ toc::[]
...
+[NOTE]
+====
+Managed services options are also available if the RHOAS Operator is installed.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc:24]: Done! No, it is not a GUI element hence we don't need to keep it bold.

File: applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc [hunk 13]
```diff
@@ -7,14 +7,37 @@ toc::[]
...
-* *Container Image*: Use existing images from an image stream or registry to deploy it on to {product-title}.
-* *From Dockerfile*: Import a dockerfile from your Git repository to build and deploy an application.
-* *YAML*: Use the editor to add YAML or JSON definitions to create and modify resources.
-* *From Catalog*: Explore the *Developer Catalog* to select the required applications, services, or source to image builders and add it to your project.
-* *Database*: See the *Developer Catalog* to select the required database service and add it to your application.
-* *Operator Backed*: Explore the *Developer Catalog* to select and deploy the required Operator-managed service.
-* *Helm Chart*: Explore the *Developer Catalog* to select the required Helm chart to simplify deployment of applications and services.
+* *Getting Started resources*: Use these resources to help you get started with the Console. You can choose to hide the header using the *Options* menu {kebab}.
+** *Creating applications using sample*: Use the existing code samples to get started with creating applications on the {product-title}.
```
Comment (jerolimov) [inline applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc:11]:
```suggestion
** *Creating applications using samples*: Use the existing code samples to get started with creating applications on the {product-title}.
```

File: applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc [hunk 14]
```diff
@@ -7,14 +7,37 @@ toc::[]
...
 
-* *From Git*: Use this option to import an existing codebase in a Git repository to create, build, and deploy an application on {product-title}.
-* *Container Image*: Use existing images from an image stream or registry to deploy it on to {product-title}.
-* *From Dockerfile*: Import a dockerfile from your Git repository to build and deploy an application.
-* *YAML*: Use the editor to add YAML or JSON definitions to create and modify resources.
-* *From Catalog*: Explore the *Developer Catalog* to select the required applications, services, or source to image builders and add it to your project.
-* *Database*: See the *Developer Catalog* to select the required database service and add it to your application.
-* *Operator Backed*: Explore the *Developer Catalog* to select and deploy the required Operator-managed service.
-* *Helm Chart*: Explore the *Developer Catalog* to select the required Helm chart to simplify deployment of applications and services.
+* *Getting Started resources*: Use these resources to help you get started with the Console. You can choose to hide the header using the *Options* menu {kebab}.
```
Comment (jerolimov) [inline applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc:10]: On the page it is called "Getting started resources" with lower s at started. Should we call it "Developer Console" at the end? Up to you

```suggestion
* *Getting started resources*: Use these resources to help you get started with Developer Console. You can choose to hide the header using the *Options* menu {kebab}.
```

File: modules/odc-using-the-developer-catalog-to-add-services-or-components.adoc [hunk 15]
```diff
@@ -9,11 +9,8 @@ You use the Developer Catalog to deploy applications and services based on Opera
 
 .Procedure
 
-. In the *Developer* perspective, navigate to the *+Add* -> *From Catalog* to view all the available services in the *Developer Catalog*.
-. Under *All Items*, select the kind of service or the component you need to add to your project. For this example, select *Databases* to list all the database services and then click *MariaDB* to see the details for the service.
-+
-.Developer Catalog
-image::odc_devcatalog_mariadb.png[]
+. In the *Developer* perspective, navigate to the *+Add* view and from the *Developer Catalog* tile, click *All Services* to view all the available services in the *Developer Catalog*.
+. Under *All Services*, select the kind of service or the component you need to add to your project. For this example, select *Databases* to list all the database services and then click *MariaDB* to see the details for the service.
```
Comment (jerolimov) [inline modules/odc-using-the-developer-catalog-to-add-services-or-components.adoc:13]: ???
```suggestion
. Under *All Services*, select the kind of service or the component you need to add to your project. For this example, select *Databases* to list all the database services and then click *MariaDB* to see the details for this service.
```

File: applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc (hunk 13 above)
Comment (HarshCasper) [reply applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc:11]: Done!

File: applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc (hunk 14 above)
Comment (HarshCasper) [reply applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc:10]: Done!

**Review by christoph-jerolimov — APPROVED** [review]
/lgtm

**Review by jc-berger — APPROVED** [review]
Great work, Harsh! It's consistently in active voice and user focused. 

LGTM from docs point of view :)

**Pull request merged**